<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RestoreDataStateForm_template</class>
 <widget class="QDialog" name="RestoreDataStateForm_template">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>526</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the dialog to restore a data state">Restore data state dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget_2">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="restoreDataStateListWidgetLabel">
         <property name="text">
          <string comment="The list of the saved states">Saved states</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="restoreDataStateListWidget"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="restoreDataStateDescriptionLabel">
         <property name="text">
          <string>State description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="restoreDataStatePlainTextEdit"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="memoryConsumptionLabel">
     <property name="text">
      <string>Memory consumption for saving these states</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RestoreDataStateForm_template</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RestoreDataStateForm_template</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
